### PR TITLE
Update contact screen

### LINKED ISSUE #1109: Contact screen mockup
If we're feeling unclear about how to incorporate the list of cafes into this screen, it may be a good time to play with some ideas using mock ups.

### LINKED ISSUE #1160: Contact screen rendering not safe for multiple screen resolutions
via @carsonfarmer 

IMG_2416

### PR DESCRIPTION
**Pull request by undercase**
Fixes #1160. Fixes #1109. Updates the contact screen to render safely for multiple screen resolutions and changes its layout to reflect more information than before using a tab view.

### CODE REVIEW

**Comment by asutula** [comment]
This on too. Let's get it updated, tests passing, and merged!

**Comment by undercase** [comment]
This is ready to be reviewed.

File: package.json [hunk 1]
```diff
@@ -26,61 +28,61 @@
...
     "react-native-fs": "^2.11.17",
-    "react-native-gesture-handler": "^1.2.1",
+    "react-native-gesture-handler": "^1.3.0",
     "react-native-hyperlink": "0.0.14",
     "react-native-image-pan-zoom": "^2.1.11",
     "react-native-image-picker": "^0.27.1",
     "react-native-modal": "^6.5.0",
     "react-native-push-notification": "^3.1.1",
     "react-native-qrcode": "^0.2.7",
+    "react-native-reanimated": "^1.0.1",
```
Comment (asutula) [inline package.json]: I don't see this used anywhere, you should remove it and unlink it

Comment (undercase) [reply package.json]: It's installed because [this component](https://github.com/jemise111/react-native-swipe-list-view) relies on it.

**Comment by undercase** [comment]
Anything I can do to get this ready for merge?

**Comment by asutula** [comment]
Nice, I'll go through it again this morning. Just need to get and sdk update done first, then this.

File: package.json (hunk 1 above)
Comment (asutula) [reply package.json]: Ok but I don't see `react-native-swipe-list-view` used anywhere either

Comment (undercase) [reply package.json]: Wait that link is wrong, it’s [this library](https://github.com/react-native-community/react-native-tab-view) that depends on it (and that one is definitely used haha).

**Pull request merged**